src/tests/getOneAccom.test.js

```diff
@@ -0,0 +1,44 @@
+import Chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../index';
+
+Chai.use(chaiHttp);
+Chai.should();
+
+describe('Barefoot nomad should allow multi city trips', () => {
```

Comment:
```suggestion
describe('Get one accommodation tests', () => {
```
@AbdoulNuru This description of test suite ```Barefoot nomad should allow multi city trips``` is ambigous. Right?

Reply:
@hakoemmy Thanks, that is a mistake